### PR TITLE
AC-7833, AC-7833 and AC-7834: Fix impact Api tests and get them to run as part of accelerate test suite

### PR DESCRIPTION
**Pull request by shanbady**
See associated PR for instructions:
masschallenge/accelerate#2635

### CODE REVIEW

**Comment by pbx** [comment]
(As I noted on the accelerate PR, the test run worked for me, so my approval here is just waiting on on a decision about my cleanup idea.)

File: accelerator/tests/contexts/analyze_judging_context.py [hunk 1]
```diff
@@ -7,8 +7,14 @@
...
 from accelerator.models import (
     JUDGING_FEEDBACK_STATUS_COMPLETE,
-    JudgeApplicationFeedback,
 )
+from accelerator.apps import AcceleratorConfig
+import swapper
+
+
+JudgeApplicationFeedback = swapper.load_model(
+    AcceleratorConfig.name,
```
[the remark this answers is not in the document]
Comment (pbx) [reply accelerator/tests/contexts/analyze_judging_context.py:16]: (We've discussed this and agreed that factoring this out deserves its own ticket -- there are over 100 `AcceleratorConfig` references.)

[the remark this answers is not in the document]
Comment (shanbady) [reply accelerator/tests/contexts/analyze_judging_context.py:16]: created s spin-off ticket to clean these up: https://masschallenge.atlassian.net/browse/AC-7856

[the remark this answers is not in the document]
Comment (pbx) [reply accelerator/tests/contexts/analyze_judging_context.py:16]: https://masschallenge.atlassian.net/browse/AC-7856

**Pull request merged**